src/main/java/io/xstefank/wildlfy/bot/TriagePullRequestProcessor.java

```diff
@@ -1,21 +1,55 @@
 package io.xstefank.wildlfy.bot;
 
 import io.quarkiverse.githubapp.ConfigFile;
+import io.quarkiverse.githubapp.GitHubClientProvider;
+import io.quarkiverse.githubapp.GitHubConfigFileProvider;
 import io.quarkiverse.githubapp.event.PullRequest;
+import io.quarkus.runtime.Startup;
 import io.xstefank.wildlfy.bot.config.WildFlyConfigFile;
 import io.xstefank.wildlfy.bot.config.WildFlyConfigFile.WildFlyRule;
 import io.xstefank.wildlfy.bot.config.util.Matcher;
+import jakarta.annotation.PostConstruct;
+import jakarta.inject.Inject;
 import org.jboss.logging.Logger;
+import org.kohsuke.github.GHAppInstallation;
 import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GitHub;
 import org.kohsuke.github.GHPullRequest;
 
 import java.io.IOException;
 import java.util.Set;
 import java.util.TreeSet;
 
+@Startup
 public class TriagePullRequestProcessor {
     private static final Logger LOG = Logger.getLogger(TriagePullRequestProcessor.class);
 
+    @Inject
+    GitHubClientProvider clientProvider;
+
+    @Inject
+    GitHubConfigFileProvider fileProvider;
+
+    @PostConstruct
+    void checkConfiguration() {
+        try {
+            for (GHAppInstallation installation : clientProvider.getApplicationClient().getApp().listInstallations()) {
+                GitHub app = clientProvider.getInstallationClient(installation.getId());
+                for (GHRepository repository : app.getInstallation().listRepositories()) {
+                    WildFlyConfigFile wildflyBotConfigFile = fileProvider.fetchConfigFile(repository, "wildfly-bot.yml", ConfigFile.Source.DEFAULT, WildFlyConfigFile.class).get();
```

Comment:
We should put the name of the config to some shared constant somewhere so it doesn't need to be typed in string everywhere and prone to errors.

Reply:
I have opened an issue for this please refer to #53